jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/tx/DisposeSessionTransactionSynchronization.java

```diff
@@ -42,7 +45,7 @@ public void afterCompletion(int status) {
 	    try {
 	        manager.disposeRuntimeEngine(runtime);
 	    } catch (Throwable e) {
-	        // catch exception as it's only clean up and should not affect runtime
+	        logger.warn("Could not dispose runtime engine", e);
```

Comment:
could we make it debug instead, there could be number of cases where this will make logs looks bit scary while it should not be seen as any serious issue but might be worth looking into while running in debug, wdyt?